hazi.css

```diff
@@ -0,0 +1,193 @@
+.basic {
+  padding: 5px;
+  margin: 10px;
+  background-color: rgb(163, 33, 33);
+}
+
+h1:first-letter{
+  font-weight: bold;
+  font-size: 48px;
+  color: #7ab4d8;
+}
+
+.header_basic{
+  font-family: "Lucida Handwriting", cursive;
+  color: lightskyblue;
+}
+
+h1{
+  font-size: 32px;
+  text-shadow: 2px 2px #7a94a4;
+}
+
+h2{
+  font-size: 28px;
+  margin: 10px
+}
+
+
+h3{
+  font-size: 26px;
+  margin: 15px;
+}
+
+h4{
+  font-size: 24px;
+  margin: 20px;
+}
+
+p{
+  margin: 15px;
+  font-size: 16px;
+  color:cadetblue;
+}
+
+.link_basic{
+  font-size: 14px;
+  line-height: 5;
+}
+
+.highlight{
+  background-color: yellow;
+}
+
+details{
+  font-size: 14px;
+  color:cadetblue;
+}
+
+.smaller{
+  font-size: 12px;
+  color:cadetblue;
+}
+
+ul{
+  font-size: 16px;
+  color:cadetblue;
+  list-style-type: square;
+}
+
+li{
+  padding: 5px;
+  margin: 15px;
+}
+
+details > ul {
+  font-size: 12px;
+  font-style: italic;
+}
+
+details > li {
+  padding: 1px;
+  margin: 20px;
+}
+
+img{
+  margin: 15px;
+  width: 250px;
+  float: left;
+}
+
+.jobb_kepek{
+  width: 250px;
+  float: right;
+}
+
+.hatter{
+  background-image: url("bracsa_hatter.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.bracsa_hegedu_ossze{
+  width: 100px;
+}
+
+.footer_nav{
+  backdrop-filter: invert(100%);
+  border-radius: 15px;
+}
+
+
+div{
+  margin: 0 auto;
+  text-align: justify;
+  width: 1000px;
+  padding: 5px;
+}
+
+.div_border{
+  border-radius: 15px;
+  border: 1px solid aqua;
+}
+
+select{
+  appearance: none;
+}
+.rejtett{
```

Comment:
A blokkok között érdemes egyetlen üres sor szünetet tartani. Van, ahol kettő is van.